.github/workflows/evm.yml

```
@@ -6,6 +6,8 @@ on:
   push:
     branches:
       - main
+    tags:
+      - "v*+evm"
```

actionlint (1.7.10)

[error] invalid glob pattern. unexpected character '+' while checking special character + (one or more). the preceding character must not be special character. note: filter pattern syntax is explained at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

(glob)